osu.Game/Overlays/OverlayRulesetTabItem.cs

```diff
@@ -48,15 +46,14 @@ public OverlayRulesetTabItem(RulesetInfo value)
                 {
                     AutoSizeAxes = Axes.Both,
                     Direction = FillDirection.Horizontal,
-                    Spacing = new Vector2(3, 0),
-                    Child = text = new OsuSpriteText
+                    Spacing = new Vector2(4f, 0),
```

Comment:
What's with the random `f` number type spec here? Why is it half-and-half, rather than raw integers (as it was before), or `new Vector2(4f, 0f)` even? Not a huge deal but really weird change out of nowhere...

Reply:
I mean, sure... I tend to type any number followed by `f` whenever it's a vector.

Reply:
Why was this conversation resolved? I don't see any new commits that would address the review? (Also generally, bad form to resolve conversations if you're not a reviewer, as has been stated several times by now)

Reply:
Uh, had committed but got interrupted with something else and forgot to push.